ADR/adr-0006-names-labels.md

```diff
@@ -0,0 +1,76 @@
+# ADR-0006 Log Conventions
+
+Date: 2022-November-11
+
+## Status
+
+Accepted
+
+## Approvers
+
+* Ann Marie Fred
+* Gorkem Ercan
+
+## Reviewers
+
+## Context
+
+We need log conventions to make our offering easier to operate and maintain. These will also make it possible for us to create queries that cross components in our logs.
+
+## Decision: Log conventions
+
+In our controller logs, we will use structured, JSON-formatted audit log messages with key-value pairs as described below.
+
+Fluentd and Fluent Bit annotate log messages with the following information automatically: `namespace_name`, `container_name`, `pod_name`, `container_image`, `pod_ip`, `host`, `hostname`, `namespace_labels`, `message`, `level`, `time`, and more. The cluster, node, pod and container names are also part of the log stream name.  For example:
+  Under /aws/containerinsights/`Cluster_Name`/application
+  - Fluent Bit optimized configuration sends logs to `kubernetes-nodeName`-application.var.log.containers.`kubernetes-podName`_`kubernetes-namespace`_`kubernetes-container-name`-`kubernetes-containerID`
+  - Fluentd sends logs to `kubernetes-podName`_`kubernetes-namespace`_`kubernetes-containerName`_`kubernetes-containerID`
+
+For more details on Fluentd vs. Fluent Bit logs, see [Set up Fluent Bit as a DaemonSet to send logs to CloudWatch Logs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs-FluentBit.html).
+
+It is best to avoid using the same key as the log collectors in our controller logs, to avoid confusion.
+
+### 1. When did it happen? 
+
+Timestamps will be in UTC/ISO-8601 format.
+
+### 2. What happened?
+
+Use the key `action` with possible values `VIEW, ADD, UPDATE, DELETE`.
```

Comment:
Current logs in application-service:
```
ERROR controllers.Component Unable to fetch the created webhook el-cool-catalog-hqw4, retrying
```
```
INFO controllers.Component Starting reconcile loop for mkovarik/cool-catalog-hqw4
```

Which action should be added to such logs?